sources/documents/2_RTB/Verbali/Esterni/VerbaleEsterno_2024-05-06/capitolo-riunione.tex

```diff
@@ -0,0 +1,59 @@
+\section{Riunione}
+\subsection{Ordine del giorno}
+\begin{itemize}
+	\item Discutere delle tecnologie per lo sviluppo della web app;
+	\item Esporre studio preliminare delle tecnologie lato back-end;
+	\item Valutare l'ampliamento dei casi d'uso;
+\end{itemize}
+
+\subsection{Argomenti e temi dell'incontro}
+
+\subsubsection{Tecnologie per lo sviluppo della web app}
+
+\textbf{Domanda:} Tra le diverse tecnologie solitamente indicate per lo sviluppo di un'applicazione web (quali Node.js, Flask etc.), abbiamo individuato il \glossario{framework} \glossario{Streamlit}. 
+Questo permette uno sviluppo \glossario{front-end} più celere rispetto alle alternative, seppur conceda meno spazio alla personalizzazione. 
+Secondo lei, potrebbe essere una scelta opportuna?
+
+\textbf{Risposta:} La \glossario{Proponente} non è familiare col framework Streamlit, ma ne approverebbe l'utilizzo a seguito di una presentazione informativa da parte del gruppo in cui si spiegano approfonditamente, ovvero tramite paragoni con gli altri framework, le motivazioni di tale scelta.
+
+\subsubsection{Esposizione studio preliminare delle tecnologie lato \glossario{back-end}}
+
+\textbf{Descrizione:} Il \glossario{Programmatore} espone il lavoro di studio sulla libreria \glossario{txtai}, con particolare enfasi riguardo la componente di indicizzazione e ricerca. 
+Pone poi l'attenzione sulla necessità o meno di fasi di pre-processing delle queries, nel tentativo di semplificare la ricerca semantica per la LLM. 
```

Comment:
Il programmatore espone i risultati conseguiti durante lo studio della libreria \glossario{txtai}, con particolare enfasi sulla componente di \glossario{indicizzazione} e ricerca semantica. 
Pone poi l'attenzione sulla necessità o meno delle fasi di pre-processing della richiesta dell'utente, nel tentativo di semplificare la ricerca semantica. 
L'idea del team sarebbe di processare la richiesta in linguaggio naturale per \glossario{tokenizzare} solo le componenti semantiche ritenute fondamentali. Allo stesso modo, sarebbe prevista un'elaborazione del \glossario{dizionario dati} in formato \glossario{JSON} per renderlo simile al linguaggio naturale, cosicché i \glossario{modelli}, allenati su quest'ultimo, riescano ad estrarne con più facilità il significato.